modules/wmi/testdata/v0.20.0/metrics.txt

```diff
@@ -103,6 +103,188 @@ process_start_time_seconds 1.667508736e+09
 # HELP process_virtual_memory_bytes Virtual memory size in bytes.
 # TYPE process_virtual_memory_bytes gauge
 process_virtual_memory_bytes 3.5569664e+07
+# HELP windows_adcs_challenge_response_processing_time_seconds Last time elapsed for challenge response
+# TYPE windows_adcs_challenge_response_processing_time_seconds gauge
+windows_adcs_challenge_response_processing_time_seconds{cert_template="Administrator"} 0
+windows_adcs_challenge_response_processing_time_seconds{cert_template="DirectoryEmailReplication"} 0
```

Comment:
Let's just have 1-2 cert templates. No need to have all of them, it will allow us to have (much) less lines in wmi_test.go

Reply:
All right, I kept `Administrator`, `DomainController` and `_Total` that is the sum of two previous.